test_scripts/API/MainMenuUpdating/001_dynamic_update_capabilities_success.lua

```diff
@@ -0,0 +1,136 @@
+---------------------------------------------------------------------------------------------------
+-- Description:
+-- HMI sends capability with DynamicUpdateCapabilities parameter
+
+-- Pre-conditions:
+-- a. HMI and SDL are started
+-- b. appID is registered and activated on SDL
+-- c. appID is currently in Background, Full or Limited HMI level
+
+-- Steps:
+-- HMI sends DISPLAYS system capability update with Dynamic Update capabilities
+
+-- Expected:
+-- Mobile receives capability update with corrent params.
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/Smoke/commonSmoke')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local addSubMenuParams = {
```

Comment:
addSubMenuParams, setGlobalPropertiesParams and successResponse are unused and can be safely removed